docker/requirements/requirements.0.18.0b1.txt

```diff
@@ -17,7 +17,7 @@ decorator==4.4.2
 docutils==0.15.2
 google-api-core==1.16.0
 google-auth==1.16.1
-google-cloud-bigquery==1.24.0
+google-cloud-bigquery==1.25.0
```

Comment:
does 1.25 specifically add support for policy tags? We had some issues with recent releases of a couple of Google libraries, so I just wanted to double check that bumping this version number is required for this change

Reply:
Hey @drewbanin, support for policy tags was added in 1.25.0 which is why I updated the version
https://googleapis.dev/python/bigquery/latest/changelog.html